### ISSUE TITLE
dependency mime ~1.3.4 has a ReDoS vulnerability

### ISSUE THREAD

**Issue by pmuellr** [isaacs/st#90]
see: https://snyk.io/vuln/npm:mime:20170907

Hopefully this is a simple update of the dependency, will take a look soon.

**Comment by dyaa** [comment]
As nodesecurity's Remediation

Please upgrade to version 2.0.3 or greater

https://nodesecurity.io/advisories/535

**Comment by pmuellr** [comment]
> Please upgrade to version 2.0.3 or greater

The PR ref'd here - https://github.com/isaacs/st/pull/91 - only upgrades to `~1.4.1` since presumably there's less API surface change between this and 2.0.3 (without having to investigate too deeply).

The NSP page ref'd does say to upgrade to >= 2.0.3 in the big "Remediation" section, but in the top left also says: 

```
Vulnerable: < 1.4.1 || > 2.0.0 < 2.0.3
Patched: >= 1.4.1 < 2.0.0 || >= 2.0.3
```
